stable/postgresql/README.md

```diff
@@ -58,6 +58,8 @@ The following table lists the configurable parameters of the PostgreSQL chart an
 | `existingSecret`           | Use Existing secret for Admin password          | `nil`                                                      |
 | `postgresConfig`           | Runtime Config Parameters                       | `nil`                                                      |
 | `pgHbaConf`                | Content of pg\_hba.conf                         | `nil (do not create pg_hba.conf)`                          |
+| `initdbdSecret`            | Secret containing initdb.d files                | `nil`                                                      |
+| `initdbdMountPath`         | initdb.d files mount path                       | `/docker-entrypoint-initdb.d`                              |
```

Comment:
Maybe state that it is related to initdbSecret and has effect only if enabled?

Reply:
Added. Only used if initdbdSecret is set